README.md

```diff
@@ -1,110 +1,104 @@
-
-- Notify Type: http
-- Event Type: artifact pushed
-- Endpoint URL: `<acceleration service address>`/api/v1/conversions
-- Auth Header: `<configured in acceleration service>`
-
-2. Add robot account
-
-Administration -> Robot Accounts -> `+ NEW ROBOT ACCOUNT`, add robot account with following fields:
-
-- Expiration time: `<by your choice>`
-- Reset permissions: select `Push Artifact`, `Pull Artifact`, `Create Tag`
-
-Generate a base64 encoded auth string using robot account name and secret like this:
-
-``` shell
-$ printf '<robot-name>:<robot-secret>' | base64
-```
-
-### Configuration (Acceleration service side)
-
-1. Copy template config file
-
-nydus:
-
-``` shell
-$ cp misc/config/config.nydus.yaml misc/config/config.yaml
-```
-
-estargz:
-
-``` shell
-$ cp misc/config/config.estargz.yaml misc/config/config.yaml
-```
-
-2. Edit config file
-
-Following comments in `misc/config/config.yaml`, ensure base64 encoded auth string and webhook auth header be configured correctly.
-
-### Boot service
-Boot daemon to serve webhook request in PUSH_ARTIFACT event
-
-``` shell
-$ ./acceld --config ./misc/config/config.yaml
-```
-
-### Test image conversion
-
-1. Trigger image conversion
-
-We can either push an image to Harbor:
-
-``` shell
-$ docker push <harbor-service-address>/library/nginx:latest
-```
-
-Or convert an existing image manually using the `accelctl` CLI tool:
-
-``` shell
-$ ./accelctl task create <harbor-service-address>/library/nginx:latest
-$ ./accelctl task list
-```
-
-2. Got converted image
-
-If everything is ready, we can see the log output in acceleration service like this:
-
-``` shell
-INFO[2021-09-17T05:43:49.783769943Z] Version: ecbfefc312ea78d81eb895ef7768a60caa30a281.20210917.0543
-INFO[2021-09-17T05:43:49.786146553Z] [API] HTTP server started on 0.0.0.0:2077
-INFO[2021-09-17T05:44:01.82592358Z] received webhook request from 192.168.1.1:46664  module=api
-INFO[2021-09-17T05:44:01.826490232Z] POST /api/v1/conversions 200 552.695µs 392>5bytes 192.168.1.1  module=api
-INFO[2021-09-17T05:44:01.830893103Z] pulling image 192.168.1.1/library/nginx:latest  module=converter
-INFO[2021-09-17T05:44:01.916123236Z] pulled image 192.168.1.1/library/nginx:latest  module=converter
-INFO[2021-09-17T05:44:01.916175337Z] converting image 192.168.1.1/library/nginx:latest  module=converter
-INFO[2021-09-17T05:44:04.317992888Z] converted image 192.168.1.1/library/nginx:latest-nydus  module=converter
+### GETTING STARTED
+
+#### Get Harbor
+
+If you don't have a public harbor service, you can deploy a local harbor service.
+Please see `Harbor` official [documentation](https://goharbor.io/docs/latest/install-config).
+
+#### Get binaries from release page
+
+Currently, Acceleration Service includes the following tools:
+
+- An `accelctl` tool to manage acceleration service and can do conversion in one-time mode.
+- An `acceld` daemon to provide a general service to support image acceleration
```

Comment:
An `acceld` daemon to work as an HTTP service to handle image conversion requests from harbor or `accelctl`.